### ISSUE TITLE
Rule S00a test data discrepancy

### ISSUE THREAD

**Issue by mhagemann-eab** [ut-effectiveness/utValidateR#13]
``` r
library(utValidateR)
testdf <- get_test_data(file = "student")
#> Warning in check_expected_values(., colname = expected_value_column): The following rows of test data have bad expected values and were removed: 62, 63, 64, 65, 66
#> Warning in check_rule_names(., checklist = checklist, colname = rule_name_column): The following rows of test data have bad rule names and were removed: 25, 42, 43
knitr::kable(compare_rule_output("S00a", testdf = testdf))
```

| csv_row | rule | description            | expr                                       | student_id | term_id | expected | actual |
|--------:|:-----|:-----------------------|:-------------------------------------------|:-----------|:--------|:---------|:-------|
|      28 | S00a | duplicate student_id’s | !is_duplicated(cbind(student_id, term_id)) | 592029     | 202240  | fail     | pass   |

<sup>Created on 2022-09-20 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>

**Comment by mhagemann-eab** [comment]
Essentially the same issue as #12 but with student_id instead of ssn

**Comment by andreabr-dixie** [comment]
Made a change in the csv file, it should fail now.

**Comment by mhagemann-eab** [comment]
This one and S00b are still failing. I see that they have duplicated values for student_id/ssn, but the term_ids are different across the duplicated values so the rule passes. I assume this is the desired behavior--we can have the same student_id for different terms, correct? Note that if you change the term_id to be identical for the duplicated student_ids then the rule will fail for both rows of the data that have the duplicates, so you'll need to make sure the expected value and rule columns reflect this. I'd recommend adding new rows to contain the duplicates, rather than putting the duplicate values into one of the "pass" rows

**Comment by andreabringhurst** [comment]
I don't think this has been resolved. I was creating a file for bad data so we could test if the rule was working. We will need to determine if I need to go in and work on this file.

**Comment by andreabringhurst** [comment]
I think I can get this closed quickly if I can find out where the file is that I need to make a change in.
